### DISCUSSION TITLE
1.5.1.5

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#582] [Обновления]
## Список изменений
Большинство изменений внесены @SashaXser в https://github.com/ilyhalight/voice-over-translation/pull/576 и https://github.com/ilyhalight/voice-over-translation/pull/581

- Исправлено определение языка на главной странице ютуба
- Исправлен двойной вызов перевода на /embed/
- Исправлено двойное определение videoID (иногда не определялась длительность видео)
- Исправлена невидимая кнопка на /embed/
- Исправлен сброс перевода при смене качества видео
- Улучшена очистка описания на YouTube
- Добавлен полифил для GM_addStyle
- Другие мелкие правки в коде

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.1.4...1.5.1.5

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.1.5'>1.5.1.5</a>.</em>

**Comment by shyp00per** [comment]
Здравствуйте! Подскажите пожалуйста, как добавить такие сайты как http://seasonvar.ru/ http://fanserials.vip и другие подобные ресурсы? Иногда очень нужны субтитры.. Благодарю.
